### ISSUE TITLE
Better management of pop-up windows

### ISSUE THREAD

**Issue by jailby** [CUTR-at-USF/usf-mobullity#35]
The fact to have a pop-up for the itineraries generate usability issues. For example, when clicking on the button "Layer" or "Trip Planner" their respective box appear under the pop up and the user doesn't know about that. The user need to close or reduce the itineraries pop-up to notice the box he/she opened.

Change the pop-up for itineraries to a different box would enhance the usability of the interface.

1)The best solution is to have the itineraries in the same box than the trip planner box but under the option to plan a trip. It seems, to me, the most usable solution.
There is also other solution:
2) (easiest to implement) A solution would be to make the default area of this pop-up different. For example under the Trip Planner box. My concern with this solution is the appearance aspect of it.
3) Having a third button after "Trip Planner" and "Layer" which would be "Itineraries" seem like a reasonable solution.

Example for other trip planners:
http://opentripplanner.usf.edu/#/submit&fromPlace=28.059608,-82.415609&toPlace=28.063262,-82.413055&mode=WALK&min=QUICK&maxWalkDistance=840&walkSpeed=1.341&time=1:40pm&date=7/15/2015&arriveBy=false&itinID=1&wheelchair=false&preferredRoutes=&unpreferredRoutes=

http://ride.trimet.org/#/submit&fromPlace=475 NW GLISAN ST::45.526077,-122.675835&toPlace=NE Lincoln Ct @ NE Lincoln St::45.523136,-122.968890&mode=WALK&min=QUICK&maxWalkDistance=840&time=1:44 pm&date=7/15/2015&arriveBy=false&itinID=1&wheelchair=&preferredRoutes=&unpreferredRoutes=

**Comment by barbeau** [comment]
I agree that this could be improved.  The biggest consideration for me, though, is how this looks/works on mobile as well as desktop.

@jmfield2 Any ideas for improvement here, considering the implementation constraints we have?

**Comment by jmfield2** [comment]
It seems like the quick/dirty solution to this would be to make sure attempting to activate a different window isn't blocked by another window.  

Perhaps minimizing all other windows first -- though this might become annoying too.

or, simply bringing the requested window above any others?

**Comment by barbeau** [comment]
Bringing requested window to front would probably be ok - although this means to get back to the map you need to manually minimize all windows.  I'd be curious to see what the interaction looks like when always minimizing all other windows first, if there is some kind of flicker effect.

**Comment by barbeau** [comment]
From my meeting with Chris Akin (USF IT Director) yesterday - a few items that fall under this issue:
1.  We decided that the MyUSF app will include icons for maps, etc. that link out to the MoBullity site.  However, we'd want to link out to the Trip Planner and Layers windows separately - in other words, we'd need the ability to default to different windows based on a URL.
2.  Tabs should be colored differently based on what pop-up window current has focus.
3.  On mobile the Layers popup that initially appears takes up most of the top half of the screen, when it could be resized to a much smaller area - see all the white space in the Layers box below:

![image](https://cloud.githubusercontent.com/assets/928045/9137576/a95361e6-3cec-11e5-8eb4-6139e3b97a4d.png)

**Comment by jmfield2** [comment]
Seems like the only sub-issues left to-do from this issue are:
1) Improvements to the Trip Planner / Itinerary view ("enhancement")
2) Tab colors
3) Make sure layers isn't too big on mobile

Also, it seems like #3 is fixed since it is sized to fit the contents. Confirm?

**Comment by jailby** [comment]
For 1) we probably need a separate issue and for 2) there is already an issue about it.

Once an issue for 1) is created we can close this issue.

**Comment by barbeau** [comment]
Resizing widget (3) still isn't fixed - if you switch to the layers tab there is still a lot of white space.  Let's move that to its own issue also.

**Comment by jailby** [comment]
I am closing this issue because it has been broken down in multiple sub-issues.

1) Improvements to the Trip Planner / Itinerary view ("enhancement") #120 
2) Tab colors #107 
3) Make sure layers isn't too big on mobile #119
